src/components/ChatEntry.css

```diff
@@ -97,4 +97,10 @@ button {
 
 .chat-entry.remote .entry-bubble:hover::before {
   background-color: #a9f6f6;
-}
\ No newline at end of file
+}
+/* .unliked{
+  display:"❤️";
+}
+.liked{
+  display:"🤍";
+} */
```

Comment:
This CSS is commented out, we should probably omit this file from our PR if we don't want our teammates to also pull this code.